exporter.go

```diff
@@ -182,12 +183,41 @@ func main() {
 	authTokens = kingpin.Flag(
 		"telemetry.auth", "List of valid bearer tokens. Defaults to none (no auth)").Strings()
 
-	log.AddFlags(kingpin.CommandLine)
+	//log.AddFlags(kingpin.CommandLine)
+	loglevel := "debug"
+	logformat := ""
+	kingpin.Flag("log.level", "Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal]").
+		Default(loglevel).StringVar(&loglevel)
+	defaultFormat := url.URL{Scheme: "logger", Opaque: "stderr"}
+	kingpin.Flag("log.format", `this is deprecated`).
+		Default(defaultFormat.String()).
+		StringVar(&logformat)
```

Comment:
Can you use https://pkg.go.dev/github.com/prometheus/common@v0.31.1/promlog to construct the logger and https://pkg.go.dev/github.com/prometheus/common@v0.31.1/promlog/flag to define the flags here?